### PR TITLE
Add the following to facilitate CI builds:

### PR DESCRIPTION
**Pull request by zaro0508**
* fastlane - command line tool to build xcode apps
* travis - configurations for Travis CI

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    # sh "your_script.sh"
+    # You can also use other beta testing services here (run `fastlane actions`)
+  end
+
+  desc "Deploy a new version to the App Store"
+  lane :release do
+    # match(type: "appstore")
+    # snapshot
+    gym(scheme: "ResearchKit") # Build your app - more options available
```
Comment (zaro0508) [inline fastlane/Fastfile]: should be "ResearchUXFactory"

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    else
+      scan
+    end
+  end
+
+  desc "Submit a new Beta Build to Apple TestFlight"
+  desc "This will also make sure the profile is up to date"
+  lane :beta do
+    # match(type: "appstore") # more information: https://codesigning.guide
+    gym(scheme: "ResearchKit") # Build your app - more options available
```
Comment (zaro0508) [inline fastlane/Fastfile]: should be "ResearchUXFactory"

File: fastlane/Appfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+app_identifier "org.researchkit.ResearchUXFactory" # The bundle identifier of your app
```
Comment (syoung-smallwisdom) [inline fastlane/Appfile]: org.sagebase ?

Comment (zaro0508) [reply fastlane/Appfile]: yep, it has been updated.

**Review by syoung-smallwisdom — APPROVED** [review]
looks good

**Pull request merged**